src/test/pythonnode_test.cpp

```diff
@@ -2260,7 +2260,7 @@ TEST_F(PythonFlowTest, FinalizePassTest) {
     ASSERT_TRUE(::google::protobuf::TextFormat::ParseFromString(pbTxt, &config));
 
     std::shared_ptr<PythonNodeResources> nodeResources = nullptr;
-    ASSERT_EQ(PythonNodeResources::createPythonNodeResources(nodeResources, config.node(0), getPythonBackend()), StatusCode::OK);
+    ASSERT_EQ(PythonNodeResources::createPythonNodeResources(nodeResources, config.node(0), getPythonBackend(), ""), StatusCode::OK);
```

Comment:
Is empty string a possible value? If createPythonNodeResources is not called with empty string as a basePath in any real scenario then this setting does not provide valueable information and might be misleading.

Reply:
It is not possible in regular use case when pbtxt is located on file system. In case of these tests every value is equally valid, since there is no actual basePath in those scenarios.